Yosemite/Yosemite/Stores/ShippingLabelStore.swift

```diff
@@ -36,17 +36,109 @@ public final class ShippingLabelStore: Store {
         }
 
         switch action {
+        case .synchronizeShippingLabels(let siteID, let orderID, let completion):
+            synchronizeShippingLabels(siteID: siteID, orderID: orderID, completion: completion)
         case .printShippingLabel(let siteID, let shippingLabelID, let paperSize, let completion):
             printShippingLabel(siteID: siteID, shippingLabelID: shippingLabelID, paperSize: paperSize, completion: completion)
         }
     }
 }
 
 private extension ShippingLabelStore {
+    func synchronizeShippingLabels(siteID: Int64, orderID: Int64, completion: @escaping (Result<Void, Error>) -> Void) {
```

Comment:
For "syncing" actions for a list of values, we usually don't return the values in the completion block because we get the values from FetchedResultsController instead of using them directly. Do you think it's still worth returning the values in the completion block in this case?

Reply:
Got it. No need to. I was just curious. 🙂